proto3-suite.cabal

```diff
@@ -222,9 +237,7 @@ test-suite tests
     , text-short >=0.1.3 && <0.2
     , transformers >=0.4 && <0.7
     , turtle
-    , vector >=0.11 && <0.14
-
-  ghc-options:         -O2 -Wall
+    , vector >=0.11 && < 0.14
```

Comment:
There's a change to add a space after the `<`; any particular reason?  (I've seen both styles.)